packages/mwp-api-proxy-plugin/.babelrc.js

```diff
@@ -0,0 +1 @@
+module.exports = require('../../babel.config.js');
```

Comment:
It looks like babel has this `rootMode` option which if you supply `upward`, will have it search parent directories until it finds the babel.config.js file, so you might be able avoid this per-package configs :

https://babeljs.io/docs/en/options#rootmode

The other option would be to have all are builds running relative to the root directory, in which case the per-package configuration might not be necessary either.

I got this info from :https://babeljs.io/docs/en/config-files#root-babelconfigjs-files

Not sure if you tried it. It's possible it doesn't work for other reasons, but wanted to share.

Reply:
Ah, cool, I've updated to use `--root-mode upward` in the `babel` calls in each package. Definitely cleaner.